watchdog/server.sls

```diff
@@ -0,0 +1,46 @@
+{%- from "watchdog/map.jinja" import server with context %}
+{%- if server.enabled %}
+
+watchdog_packages:
+  pkg.installed:
+    - name: watchdog
+
+{%- if server.kernel_module is defined %}
```

Comment:
`server.kernel.module`

Reply:
@ruzickap Please reflect these changes also in the test pillars.